runtime/compiler/env/VMJ9.h

```diff
@@ -1024,6 +1024,13 @@ class TR_J9VMBase : public TR_FrontEnd
     */
    virtual bool isDontInline(TR_ResolvedMethod *method);
 
+   /**
+    * \brief Determine whether a method is annotated with @IntrinsicCandidate.
+    * \param method method
+    * \return true if a @DontInline annotation is present, false otherwise
```

Comment:
Was this meant to be:
`* \return true if a @IntrinsicCandidate annotation is present, false otherwise`?

Reply:
Yes, thanks!